### DISCUSSION TITLE
Do you have any idea to revive the LSPatch project?

### DISCUSSION THREAD

**Discussion by woniu163** [mywalkb/LSPosed_mod#57] [General]
### Is your feature request related to a problem?/你的请求是否与某个问题相关？

Do you have any idea to revive the LSPatch project?

### Describe the solution you'd like/描述你想要的解决方案

LSPatch is a non-root Xposed framework extending from LSPosed

**Comment by mywalkb** [comment]
No, I have never used LSPatch, I started modifying LSPosed to put the thing I needed most: modifying the scope of an xposed module, from the module itself without having to use root
I do this in the little spare time I have, so it's hard to make time for a project I'm not familiar with.
